### PR TITLE
Unbreak master by fixing StreamFileDocSpec to import scaladsl._ iso io._

### PR DESCRIPTION
**Pull request by viktorklang**

### CODE REVIEW

**Comment by drewhk** [comment]
LGTM

**Comment by ktoso** [comment]
Thanks for noticing and the fix. LGTM

**Comment by ktoso** [comment]
Shall we let it validate though?

**Comment by viktorklang** [comment]
@ktoso I'd wait until master is unbroken. Then rerun validation. Then merge.

File: akka-docs/rst/scala/code/docs/stream/io/StreamFileDocSpec.scala [hunk 1]
```diff
@@ -27,7 +27,7 @@ class StreamFileDocSpec extends AkkaSpec(UnboundedMailboxConfig) {
 
   {
     //#file-source
-    import akka.stream.io._
+    import akka.stream.scaladsl._
```
Comment (viktorklang) [inline akka-docs/rst/scala/code/docs/stream/io/StreamFileDocSpec.scala]: BUG WAS HERE

**Pull request merged**